Added JDBC Type 4 Driver Access to DB2/zos

dhermanns opened the pull request:
Changed the package of the db2net Driver.
Added a db2zosnet.properties to get a jdbc type 4 access to a db2z-database.

dhermanns commented:
Great - thx!

rafalkasa commented:
@dhermanns check your email used for commit contribution on github account.
Maybe this link wilk help you https://help.github.com/articles/why-are-my-contributions-not-showing-up-on-my-profile/

This will show you as contributor in the project https://github.com/schemaspy/schemaspy/graphs/contributors

dhermanns commented:
Thanks for your info - i fixed that and added my second eMail to my
Profile...
Rafal Kasa <notifications@github.com> schrieb am Fr. 4. Aug. 2017 um 17:32:

> @dhermanns <https://github.com/dhermanns> check your email used for
> commit contribution on github account.
> Maybe this link wilk help you
> https://help.github.com/articles/why-are-my-contributions-not-showing-up-on-my-profile/
>
> This will show you as contributor in the project
> https://github.com/schemaspy/schemaspy/graphs/contributors
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/schemaspy/schemaspy/pull/127#issuecomment-320279666>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADDwlWld6pUS_37GMVGVaq52AyG1Hy_Xks5sUzmQgaJpZM4OoeqN>
> .
>
